applications/SwimmingDEMApplication/custom_utilities/fields/cellular_flow_field.h

```diff
@@ -82,7 +82,7 @@ void UnlockCoordinates(const int i_thread = 0) override;
 ///@name Input and output
 ///@{
 
-/// Turn back information as a stemplate<class T, std::size_t dim> tring.
+/// Turn back information as a stemplate<class T, std::size_t dim> string.
```

Comment:
```suggestion
```

I don't think this comment is particularly helpful and it's full typos/broken English anyway, so it's better to just get rid of it.

It's also duplicated in multiple files. @luzpaz can you please remove the others too?

Reply:
Removed in 9338a6e658a07c09f929890131a99919850ed575